.eslintrc.js

```diff
@@ -0,0 +1,91 @@
+module.exports = {
```

Comment:
I copied over the config from circulation-patron-web, which I liked.  If you have one you like, copy that over and we can discuss!